packages/manager/modules/nutanix/src/dashboard/general-info/template.html

```diff
@@ -183,7 +183,11 @@
                     </div>
                 </oui-tile-description>
                 <oui-action-menu data-compact data-placement="end">
-                    <oui-action-menu-item on-click="$ctrl.goToAddNode()">
+                    <oui-action-menu-item
+                        on-click="$ctrl.goToAddNode()"
+                        data-disabled="$ctrl.nodesDetails.length >= $ctrl.MAX_NODES_BY_CLUSTER"
```

Comment:
In this way, you compute the deactivation only one time and in one place
```suggestion
                        data-disabled="$ctrl.isMaxNodesReached"
```